tests/alive-tv/attrs/noundefaggr-disableall.srctgt.ll

```diff
@@ -0,0 +1,29 @@
+; TEST-ARGS: -disable-undef-input -disable-poison-input
```

Comment:
I don't think this should prevent padding to be poison. Padding should always be poison regardless.

Reply:
I thought about this, and I agree - padding should not convey meaningful data. They cannot be retrieved using `extractvalue`, so something is weird otherwise.
This implies that reading aggregate from the memory should always fill the padding with poison - should we update `Memory::load` to do this? Maybe suggestion to LangRef, too.

Reply:
sounds good, thanks!